### PR TITLE
bump sqlservernotebook version

### LINKED ISSUE #15007: Bump SQL Server Guide package version in ADS
- Azure Data Studio Version:
Version: 1.28.0-rc1
Commit: 124f7ca887bb12e8b0054e3a707404cdedf6b688
Date: 2021-04-05T20:28:22.005Z
VS Code: 1.51.0
Electron: 9.4.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 20.3.0

### PR DESCRIPTION
**Pull request by barbaravaldez**
This PR is for #15007

### CODE REVIEW

**Review by Charles-Gagnon — APPROVED** [review]
Changed your description not to auto-close the issue so we can bring it to ask mode since we'll want this in the next release.

**Comment by barbaravaldez** [comment]
> Changed your description not to auto-close the issue so we can bring it to ask mode since we'll want this in the next release.

Thank you! The issue was approved by Ken yesterday, so once I merge this pr to main, I'll create a port request.

**Pull request merged**
